### DISCUSSION TITLE
Commit message template when moving patch into a new commit

### DISCUSSION THREAD

**Discussion by ekans** [jesseduffield/lazygit#2328] [Q&A] [answered]
When I move a patch into new commit, a commit message starting with "Split from" is used for the new message.

On my project a git hook checks the format of the commit message so the commit fails. To solve this, I have to commit myself with a message following the project pattern and continue the rebase.

I haven't found anything in the documentation but just in case: is it possible to configure a message template for this case?

Thanks for your help!

**Comment by mark2185** [comment, marked as answer]
It unfortunately isn't, and I think (may be wrong tho) that there's a feature request for prompting the user for a commit message when commiting a patch, but if you look for it and fail to find it, please turn this discussion into one.

**Reply by ekans** [reply]
There is a feature request (from you 😄): https://github.com/jesseduffield/lazygit/issues/1963

**Reply by mark2185** [reply]
Great! We'll see if the author is maybe able to finish up the PR to put this matter to bed already :)
